dags/sources/tasks/business_logic/source_data_normalize.py

```diff
@@ -53,11 +53,22 @@ def source_data_normalize(
     # Appliquer la transformation à la colonne d'origine et stocker le résultat
     # dans la colonne de destination
     for transformation in column_transformations:
-        function_name = transformation["transformation"]
-        df[transformation["destination"]] = df[transformation["origin"]].apply(
-            TRANSFORMATION_MAPPING[function_name]
-        )
-        df.drop(columns=[transformation["origin"]], inplace=True)
+        if "origin" in transformation and "destination" in transformation:
```

Comment:
Voir pour re-créer plusieurs listes, une par opération : 
- rename 
- transformation (apply) 

```suggestion
        rename_transformations = filter(lambda x: "transformation" in x, column_transformations)
        if "origin" in transformation and "destination" in transformation:
```

<img width="641" alt="image" src="https://github.com/user-attachments/assets/2d481e1d-091b-40b7-88b1-9ad3b73c6457">

Reply:
Ou voir pour utiliser une partition https://more-itertools.readthedocs.io/en/stable/api.html#more_itertools.partition